Update README.md (#705)
OpenMined/PyGrid: examples/README.md

 - **/model-centric** - Go here if you want to use PyGrid plattform from a model-centric perspective. See this [link](https://github.com/OpenMined/PyGrid#model-centric-fl) for more detail (_comming soon_).

## Running the examples

> Make sure you have [docker-compose](https://docs.docker.com/compose/install/) installed. We have prepared some images with everything installed that will help you to run the tutorials.

Reviewer: ```suggestion
Make sure you have [docker-compose](https://docs.docker.com/compose/install/) installed. We have prepared docker images that will install all the dependencies and help you run the PyGrid tutorials.
```
